content/en/agent/kubernetes/daemonset_setup.md

```diff
@@ -300,10 +300,17 @@ For Agent v6.12+, short lived container logs (stopped or crashed) are automatica
 {{% /tab %}}
 {{% tab "Docker Socket" %}}
 
-By default the Agent looks every 5 seconds for new containers. Any container with a shorter duration life does not have any data collected by the Agent.
+By default, the Agent receives container updates in real time through Docker events. However, the configuration that is extracted from the container labels (Autodiscovery) is updated every 10 seconds by the Agent. Therefore, any container with a shorter life may not have any data collected by the Agent.
 
- You can override this autodiscovery interval with a shorter one by setting the `ad_config_poll_interval` parameter which correspond to the `DD_AD_CONFIG_POLL_INTERVAL` environment variable.
-The expected value is a integer in seconds.
+You can override this interval with a shorter one by setting the `ad_config_poll_interval` parameter, which is equivalent to the `DD_AD_CONFIG_POLL_INTERVAL` environment variable.
+The expected value is a integer in seconds or you can specify the unit, for example:
```

Comment:
actually the example below is not the same parameter, `DD_AD_CONFIG_POLL_INTERVAL` is the global setting for all config providers (in seconds), below is the setting for an individual config provider unfortunately you cannot use an env var for this setting as it's within a list

Reply:
So do you suggest that we recommend to update the global one or just the one from the docker provider?

Reply:
on a docker setup the easier is to go with `DD_AD_CONFIG_POLL_INTERVAL` (but you cannot go bellow the second) on a docker+kubernetes setup (and if you use labels AD) I'd recommends using the indiviudal setting on docker to avoid polling the kubelet too often (we have a cache anyway)

Reply:
Let's focus on pure docker environment, in K8s we should recommend to go through files and pod annotations.

So I'll keep the `DD_AD_CONFIG_POLL_INTERVAL` and will change the example.